include/base/NekRSStandaloneProblem.h

```diff
@@ -31,5 +31,5 @@ class NekRSStandaloneProblem : public NekRSProblemBase
 
   static InputParameters validParams();
 
-  virtual bool movingMesh() const override;
+  virtual bool movingMesh() const;
```

Comment:
I think this line can be deleted altogether - there is no longer a need to define `movingMesh()` on a problem-type class.